project/Dependencies.scala

```diff
@@ -1,15 +1,15 @@
-import sbt._
+import sbt.*
 
 object Dependencies {
 
   val compile: Seq[ModuleID] = Seq(
-    "com.typesafe"                % "config"        % "1.4.2",
-    "com.typesafe.play"          %% "play-json"     % "2.9.4",
+    "com.typesafe"                % "config"        % "1.4.3",
+    "com.typesafe.play"          %% "play-json"     % "2.10.4",
     "com.typesafe.scala-logging" %% "scala-logging" % "3.9.5",
-    "com.vladsch.flexmark"        % "flexmark-all"  % "0.62.2",
-    "org.scalatest"              %% "scalatest"     % "3.2.17",
-    "org.seleniumhq.selenium"     % "selenium-java" % "4.16.1",
-    "org.slf4j"                   % "slf4j-simple"  % "1.7.36"
+    "com.vladsch.flexmark"        % "flexmark-all"  % "0.64.8",
+    "org.scalatest"              %% "scalatest"     % "3.2.18",
+    "org.seleniumhq.selenium"     % "selenium-java" % "4.18.1",
+    "org.slf4j"                   % "slf4j-simple"  % "2.0.9"
```

Comment:
just a note from me, on webdriver factory we kept at 1.7.x of slf4j because that was what play used and for a project that used it in the test scope of a play project then it caused problems to jump a major version, but play (maybe 2.9 but definitely 3.0 is using 2.x.x now so should be no issue for services if they are using recent version of play) https://github.com/playframework/playframework/blob/49fe612503db2edde876fb7573a407521ac3cb69/project/Dependencies.scala#L54-L57 and if they aren't then probably good case for them to split stuff out a bit to be more in line with testing guidance

Reply:
Thanks for pointing out @oscarduignan :raised_hands: